kong/plugins/prometheus/exporter.lua

```diff
@@ -71,6 +75,20 @@ local function init()
                                          "Total bandwidth in bytes " ..
                                          "consumed per service/route in Kong",
                                          {"service", "route", "type"})
+
+
+  -- Hybrid mode status
+  if kong.version_num >= 2000000 then -- 200.00.00 -> 2.0.0 then
+    if kong.configuration.role == "control_plane" then
+      cp_metrics = true
+
+      clustering = require("kong.clustering")
+
+      metrics.dataplane_last_seen = prometheus:gauge("dataplane_last_seen",
+                                                "Last time data plane contacted control plane",
+                                                {"node_id", "hostname", "ip", "hash"})
```

Comment:
I'd actually suggests to metrics here:
dataplane_last_seen{node_id, hostname}
dataplane_config_hash{node_id,hostname}

This will constrain cardinality and is actually more useful from an alerting point of view.

Reply:
Done. I need to double check if the go representation of float64 is same lua, it looks same from my eye of view.